sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -30,7 +30,7 @@ public SentenceSentiment(String text, TextSentiment sentiment, SentimentConfiden
     }
 
     /**
-     * Get the text property: The sentence text.
+     * Get the sentence text property: The sentence text.
```

Comment:
Oh sorry, what I meant was this should be just "Get the sentence text property". It's still a nitpick though, so not a big deal if you don't want to change it again